### PR TITLE
Added Agileindia and In/Clojure to the list

### PR DESCRIPTION
**Pull request by ankitg12**
**Details of the conference**:

| [Agile India 2019](https://2019.agileindia.org/) | 17th - 23rd Mar | Sheraton Grand Bangalore Hotel, Bengaluru | Asia's Largest and Premier International conference on Leading Edge Software Development Methods | No |

| [IN/Clojure](https://inclojure.org/) | 11-12 Jan | Royal Orchid Hotel, Bangalore | India's annual Clojure conference | No |

### CODE REVIEW

File: year-pages/2019-conferences-list.md [hunk 1]
```diff
@@ -3,11 +3,13 @@
 | Conference | Date | Venue | Description | Scholarship |
 |------------|------|-------|-------------|-------------|
 | [React India](https://www.reactindia.io/) | 22-24 Aug | Goa | A 3 day conference with first day focussing on workshops & the next 2 days talks on things around React, React Native, GraphQL, VR & AR, Redux, Preact and more | No |
+| [Agile India 2019](https://2019.agileindia.org/) | 17th - 23rd Mar | Sheraton Grand Bangalore Hotel, Bengaluru | Asia's Largest and Premier International conference on Leading Edge Software Development Methods | No |
```
Comment (nikhita) [inline year-pages/2019-conferences-list.md]: I can't find a CFP link for this conference (or any mention of it). This repo is only accepting conferences which have CFPs, so if you could link to a call for papers (a twitter link is also ok) I am happy to accept this one.

File: year-pages/2019-conferences-list.md [hunk 2]
```diff
@@ -3,11 +3,13 @@
...
 | [RubyConf India](https://www.rubyconfindia.org/) | 20-21 Jan | Hotel Park Regis, Goa | A global event complementing other RubyConf events across the world. This year it's a single track 2 day event, focused on the Language, Framework and Tools | [Yes](https://www.rubyconfindia.org/scholarship/) |
 | [Pragma IIITA](http://pragmaconf.tech/) | 19 -20 Jan | Indian Insitute of Information Technology, Allahabad | Pragma is premier student design and developers’ conference held at IIIT-Allahabad. Attracting some of the best student developers from across India, Pragma is a 3 day extravaganza featuring a hackathon, workshops and numerous talks. | No
 | [GopherCon](https://gopherconindia.com/) | 18-19 Jan | Hotel Park Regis, Goa | Go conference in India. Go is an open source project developed by a team at Google and many contributors for the open source community | [Yes]( https://gopherconindia.com/scholarship/) |
+| [IN/Clojure](https://inclojure.org/) | 11-12 Jan | Royal Orchid Hotel, Bangalore | India's annual Clojure conference | No |
```
Comment (nikhita) [inline year-pages/2019-conferences-list.md]: s/Bangalore/Bengaluru 

It does have a scholarship: https://inclojure.org/#opgrant

**Comment by nikhita** [comment]
@ankitg12 Do you have time to rebase and address comments? Would be good to get this in. :)

**Comment by ankitg12** [comment]
Hey @nikhita - sorry, i was busy and couldn't get to it, but Agile India had CFP call (which is closed now) here - https://confengine.com/agile-india-2019/proposals . I think since it has already started, can put it in completed ones.

**Comment by nikhita** [comment]
@ankitg12 we've got a new rule for this repo that conferences should have Code of Conduct too but I can't find one for Agile India. :confused: 

If you can find a link to the CoC, feel free to add it in. :+1:

**Comment by nikhita** [comment]
Closing due to inactivity, and since the conferences have already been over.

**Pull request closed without merging**